alvr/server_io/src/openvrpaths.rs

```diff
@@ -26,6 +26,24 @@ fn openvr_source_file_path() -> Result<PathBuf> {
     }
 }
 
+pub fn openvr_settings_file_path() -> Result<PathBuf> {
```

Comment:
rename it to `steamvr_settings_file_path` (since, well, it is steamvr settings and not openvr's config file)